server/repository/src/db_diesel/name.rs

```diff
@@ -259,6 +261,12 @@ impl<'a> NameRepository<'a> {
                 query = query.filter(name_store_join_dsl::name_is_supplier.eq(is_supplier));
             }
 
+            query = match is_patient {
```

Comment:
I was going to write, why don't we just use NameType instead of `is_patient`, but is_patient is better to use, and the query logic is clean, nice !